### DISCUSSION TITLE
准备给ZBProxy过一周年！

### DISCUSSION THREAD

**Discussion by layou233** [layou233/ZBProxy#25] [General]
![1year](https://user-images.githubusercontent.com/23739827/170453036-5fd02a37-378b-4193-80a3-94b46e269012.png)  
你猜猜会发生什么？  
不知道白名单能不能赶出来……正在冲击学考……  
> 1 Year Celebration For ZBProxy Startup

**Comment by pig8086** [comment]
白名单很好用，孩子很喜欢(๑•̀ㅂ•́)و✧

**Comment by OOPSgary** [comment]
准备写个SQL的白名单 ZBProxy很好用awa

**Reply by pig8086** [reply]
河南拔智齿
